Errors when compiling the TinyUSB lbrary using version 2.3.3

### Operating System

Windows 10

### Arduino IDE version

Arduino 2.2.1

### Board

Arduino Nano ESP32

### ArduinoCore version

Arduino Nano ESP32
![image](https://github.com/adafruit/Adafruit_TinyUSB_Arduino/assets/135274154/b3572f70-621b-4c34-847d-2126ce57041b)


### TinyUSB Library version

2.3.3

### Sketch as ATTACHED TXT

![image](https://github.com/adafruit/Adafruit_TinyUSB_Arduino/assets/135274154/f2759748-419c-40f7-ab2f-2ab17ba17a65)

[Simple_USB_MIDI_test.ino.txt](https://github.com/adafruit/Adafruit_TinyUSB_Arduino/files/14031512/Simple_USB_MIDI_test.ino.txt)

### Compiled Log as ATTACHED TXT

c:\Users\Don\Arduino\libraries\Adafruit_TinyUSB_Library\src\class\cdc\cdc_host.c:107:49: error: 'CFG_TUH_CDC_FTDI_VID_PID_LIST' undeclared here (not in a function); did you mean 'CFG_TUH_CDC_FTDI'?
 static uint16_t const ftdi_vid_pid_list[][2] = {CFG_TUH_CDC_FTDI_VID_PID_LIST};
                                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                                 CFG_TUH_CDC_FTDI
c:\Users\Don\Arduino\libraries\Adafruit_TinyUSB_Library\src\class\cdc\cdc_host.c:122:51: error: 'CFG_TUH_CDC_CP210X_VID_PID_LIST' undeclared here (not in a function); did you mean 'CFG_TUH_CDC_CP210X'?
 static uint16_t const cp210x_vid_pid_list[][2] = {CFG_TUH_CDC_CP210X_VID_PID_LIST};
                                                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                                   CFG_TUH_CDC_CP210X
c:\Users\Don\Arduino\libraries\Adafruit_TinyUSB_Library\src\class\cdc\cdc_host.c:122:51: error: initializer element is not constant
c:\Users\Don\Arduino\libraries\Adafruit_TinyUSB_Library\src\class\cdc\cdc_host.c:122:51: note: (near initialization for 'cp210x_vid_pid_list[0][0]')
c:\Users\Don\Arduino\libraries\Adafruit_TinyUSB_Library\src\class\cdc\cdc_host.c:137:50: error: 'CFG_TUH_CDC_CH34X_VID_PID_LIST' undeclared here (not in a function); did you mean 'CFG_TUH_CDC_CH34X'?
 static uint16_t const ch34x_vid_pid_list[][2] = {CFG_TUH_CDC_CH34X_VID_PID_LIST};
                                                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                                  CFG_TUH_CDC_CH34X
c:\Users\Don\Arduino\libraries\Adafruit_TinyUSB_Library\src\class\cdc\cdc_host.c:137:50: error: initializer element is not constant
c:\Users\Don\Arduino\libraries\Adafruit_TinyUSB_Library\src\class\cdc\cdc_host.c:137:50: note: (near initialization for 'ch34x_vid_pid_list[0][0]')

### What happened ?

The sketch compiles clean and runs against the 2.3.2 version of the Adafruit TinyUSB Library but the updated Adafruit library in version 2.3.3 fails with the above error messages.

Reverting back to 2.3.2 and everything is working again.

### How to reproduce ?

Attempt to compile the above sketch against the 2.3.3 version of the library and see the compiler error messages.

### Debug Log

_No response_

### Screenshots

_No response_

--- a/src/class/cdc/cdc_host.c
+++ b/src/class/cdc/cdc_host.c
@@ -30,6 +30,33 @@
 // ESP32 out-of-sync
 #ifdef ARDUINO_ARCH_ESP32
 #include "arduino/ports/esp32/tusb_config_esp32.h"
+
+#ifndef CFG_TUH_CDC_FTDI_VID_PID_LIST
+// List of product IDs that can use the FTDI CDC driver. 0x0403 is FTDI's VID
+  #define CFG_TUH_CDC_FTDI_VID_PID_LIST \
+    {0x0403, 0x6001}, {0x0403, 0x6006}, {0x0403, 0x6010}, {0x0403, 0x6011}, \
+    {0x0403, 0x6014}, {0x0403, 0x6015}, {0x0403, 0x8372}, {0x0403, 0xFBFA}, \
+    {0x0403, 0xCD18}
+#endif
+
+#ifndef CFG_TUH_CDC_CP210X_VID_PID_LIST
+// List of product IDs that can use the CP210X CDC driver. 0x10C4 is Silicon Labs' VID
+  #define CFG_TUH_CDC_CP210X_VID_PID_LIST \
+    {0x10C4, 0xEA60}, {0x10C4, 0xEA70}
+#endif
+
+#ifndef CFG_TUH_CDC_CH34X_VID_PID_LIST
+// List of product IDs that can use the CH34X CDC driver
+  #define CFG_TUH_CDC_CH34X_VID_PID_LIST \
+    { 0x1a86, 0x5523 }, /* ch341 chip */ \
+    { 0x1a86, 0x7522 }, /* ch340k chip */ \
+    { 0x1a86, 0x7523 }, /* ch340 chip */ \
+    { 0x1a86, 0xe523 }, /* ch330 chip */ \
+    { 0x4348, 0x5523 }, /* ch340 custom chip */ \
+    { 0x2184, 0x0057 }, /* overtaken from Linux Kernel driver /drivers/usb/serial/ch341.c */ \
+    { 0x9986, 0x7523 }  /* overtaken from Linux Kernel driver /drivers/usb/serial/ch341.c */
+#endif
+
 #endif
 
 #include "tusb_option.h"

